docs/v1/openapi.yaml

```diff
@@ -242,6 +242,25 @@ paths:
               $ref: '#/components/schemas/StoreAuthorizationCodeRequest'
       tags:
         - issuer
+  /issuer/interactions/exchange-authorization-code:
+    post:
+      summary: Exchange authorization code from issuer oauth provider
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExchangeAuthorizationCodeResponse'
+      operationId: exchange-authorization-code-request
```

Comment:
change "exchange-authorization-code-request" to "exchange-authorization-code" for generated handler to be `ExchangeAuthorizationCode`